apps/prairielearn/elements/pl-sketch/pl-sketch.py

```diff
@@ -0,0 +1,31 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html, data):
+    return data
+
+
+def render(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    width = pl.get_integer_attrib(element, "width", 600)
+    height = pl.get_integer_attrib(element, "height", 700)
+
+    if data["panel"] != "question":
+        return ""
+
+    if len(data["raw_submitted_answers"]) == 0:
+        skp_json = ""
+    else:
+        skp_json = data["raw_submitted_answers"]
```

Comment:
The content of the sketchpad will not be directly in `data["raw_submitted_answers"]`, but in `data["raw_submitted_answers"][answers_name]` (see other comment).